src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -31,15 +34,34 @@ class DWTConfig(object):
 
     def __init__(self, screen_drywall_cutter=None):
         self.screen_drywall_cutter = screen_drywall_cutter
-        # Load the temp config if it exists, otherwise load the default config.
-        if not os.path.exists(temp_dir):
-            os.mkdir(temp_dir)
+        self.start_up()
 
-        if os.path.exists(TEMP_CONFIG_PATH):
-            self.load_temp_config()
+    def start_up(self):
+        # Load most recent config
+        most_recent_config = self.get_most_recent_config()
+        if most_recent_config:
+            self.load_config(most_recent_config)
         else:
-            self.active_config = config_classes.Configuration.default()
-            self.load_cutter(self.active_config.cutter_type)
+            # If no configs, try to load temp config
+            if os.path.exists(TEMP_CONFIG_PATH):
+                self.load_temp_config()
+            else:
+                # Otherwise, create new config based on the default configuration
+                self.active_config = config_classes.Configuration.default()
+                self.load_cutter(self.active_config.cutter_type)
+
+    @staticmethod
+    def get_most_recent_config():
+        """
+        :return: the most recently used config path
+        """
+        files = os.listdir(CONFIGURATIONS_DIR)
+
+        if not files:
+            return None
+
+        most_recent_file = max(files, key=lambda f: os.path.getmtime(os.path.join(CONFIGURATIONS_DIR, f)))
```

Comment:
might this cause issues with the time in rPi? I heard that timestamps of logentries are sometimes off. 
And I think this also doesn't work if you don't change the file. let's say:
recent.json is the last changed one according to timestamps
DWT starts with recent.json loaded
user loads some_older.json
user leaves DWT app (without changing or saving some_older.json)
user enters DWT app again => still loads recent.json because some_older.json wasn't changed.

Maybe just save the last used config somewhere in a file?